iota/api.py

```diff
@@ -543,6 +823,25 @@ class Iota(StrictIota):
     Implements the core API, plus additional wrapper methods for common
     operations.
 
+    :param AdapterSpec adapter:
+        URI string or BaseAdapter instance.
+
+    :param Optional[Seed] seed:
+        Seed used to generate new addresses.
+        If not provided, a random one will be generated.
+
+        .. note::
+            This value is never transferred to the node/network.
+
+    :param Optional[bool] testnet:
+        Whether to use testnet settings for this instance.
+        On the testnet, minimum weight magnitude is set to 9, on mainnet
+        it is 1 by default.
```

Comment:
💭 Wondering if maybe we should make this more generic, e.g., "On the testnet, minimum weight magnitude is decreased." and/or include a link to some documentation about IOTA testnet?  I feel like this info could get overlooked and end up out-of-date eventually.

Reply:
Agreed, also created an issue to unify naming: #263